src/main/java/no/nav/fo/veilarbdialog/db/dao/DialogDAO.java

```diff
@@ -166,6 +163,13 @@ public HenvendelseData opprettHenvendelse(HenvendelseData henvendelseData) {
         return hentHenvendelse(henvendelseId);
     }
 
+    @Transactional(readOnly = true)
```

Comment:
Er hoved hensikten til denne for å hjelpe fremtidig utvikling, for å hjelpe de som leser koden eller for sikkerhet? 🤔

Reply:
ytelse. med @Transactional på kalassen blir alt håndtert som en trasaction som kan inneholde writes.
For og endre det på en metode må man legge på denne